cmd/notation/key.go

```diff
@@ -48,6 +48,20 @@ func keyCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "key",
 		Short: "Manage keys used for signing",
+		Long: `Manage keys used for signing
+
+Example - Add a key to signing key list:
+  notation key add --plugin <plugin_name> --id <key_id> <key_name>
+
+Example - List keys used for signing:
+  notation key ls
+
+Example - Update the default signing key:
+  notation key set --default <key_name>
+
+Example - Delete the key from signing key list:
+  notation key rm <key_name>...
```

Comment:
“Delete” and “remove” are two concepts. We should be clear whether it is delete or remove and make it consistent both in the description and example command.

Reply:
@shizhMSFT I think "Delete" is more exact in this example. How about we change `rm` to `Delete` to keep consistency and avoid ambiguity in the command?

Reply:
Per the [difference of remove and delete](https://apps.labor.ny.gov/ux/doc/v1/design-remove-vs-delete.html#:~:text=Remove%20and%20Delete%20are%20defined,(but%20kept%20in%20existence), it seems `remove` is more exact here. Replaced `delete` with `remove`.